.github/workflows/shellspec.yaml

```diff
@@ -11,4 +11,9 @@ jobs:
         go-version: '1.24'
     - run: |
         make shellspec
-      name: Run shell/bash script unit tests with shellspec
\ No newline at end of file
+      name: Run shell/bash script unit tests with shellspec
+    - run: echo "Shell/Bash script unit tests failed. Please check the logs for details. To run the shellspec unit tests locally, please refer to ./spec/README.md"
```

Comment:
are these necessary? should be able to get results from the `make shellspec` output right?

Reply:
`make shellspec` will run all the tests but without error details. It should be helpful for people to run a specific case with details, in terms of debugging.
 And I found that many people don't know there is a readme describing the unit test. That's why I put it here too.